internal/pods/schedule.go

```diff
@@ -148,12 +148,21 @@ func (np *Scheduled) schedule() error {
 		},
 	}
 
+	// Some containers which run os like rhel/fedora runs tcpdump
+	// as specific user id "72". So it needs pods to be privileged.
+	// Also setting the runAsUser prevent the pods from starting with
+	// random user id
+	privileged := true
+	runAsUser := int64(0)
+
 	if np.Config.Scheduling.Networking == HostNetworking {
 		networkPod.Spec.Containers[0].SecurityContext = &v1.SecurityContext{
 			Capabilities: &v1.Capabilities{
 				Add:  []v1.Capability{"NET_ADMIN", "NET_RAW"},
 				Drop: []v1.Capability{"all"},
 			},
+			Privileged: &privileged,
+			RunAsUser:  &runAsUser,
```

Comment:
Please use `k8s.io/utils/pointer`:

```suggestion
			Privileged: pointer.Bool(true),
			RunAsUser:  pointer.Int64(0),
```